database/migrations/2020_02_21_230554_add_experience_work_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddExperienceWorkTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('experience_work', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('title');
+            $table->string('organization');
+            $table->string('group')->nullable();
+            $table->boolean('is_active')->nullable();
+            $table->date('start_date');
+            $table->date('end_date');
+            $table->integer('work_experienceable_id')->unsigned()->index();
+            $table->string('work_experienceable_type')->nullable();
```

Comment:
These should either both be nullable, or neither. I think in our case we want neither. We shouldn't have any free-floating experiences. (This goes for the other models as well.)